### PR TITLE
Update tox.ini.j2

### PR DESCRIPTION
**Pull request by jnsgruk**
Fixes an issue with codespell and later versions of tox

### CODE REVIEW

File: charmcraft/templates/init-simple/tox.ini.j2 [hunk 1]
```diff
@@ -40,7 +40,7 @@ deps =
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell {toxinidir}/. \
+    codespell {toxinidir} \
```
Comment (lengau) [inline charmcraft/templates/init-simple/tox.ini.j2:43]: This probably needs to be changed in `init-machine/tox.ini.j2` and `init-kubernetes/tox.ini.j2` as well.

Comment (jnsgruk) [reply charmcraft/templates/init-simple/tox.ini.j2:43]: Pretty sure they're just symlinks to here, no?

**Comment by lengau** [comment]
@jnsgruk Ope, you're right! Sorry about that - the new version of PyCharm isn't italicising symlinks anymore so I missed that.

File: charmcraft/templates/init-simple/tox.ini.j2 (hunk 1 above)
Comment (lengau) [reply charmcraft/templates/init-simple/tox.ini.j2:43]: Ope, you're right! Sorry about that - the new version of PyCharm isn't italicising symlinks anymore so I missed that.

**Review by lengau — APPROVED** [review]

**Comment by facundobatista** [comment]
This was superseded by https://github.com/canonical/charmcraft/pull/957

**Pull request closed without merging**